src/components/Verify/styles.ts

```diff
@@ -0,0 +1,49 @@
+import { Box, Typography } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { theme } from '@theme';
+
+export const WrapperContainer = styled(Box)({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  minHeight: '100vh',
+});
+
+export const CenteredContent = styled(Box)`
+  align-items: center;
+  h2 {
+    font-size: ${theme.typography.sizes.h1};
+  }
+`;
+
+export const LoadingText = styled(Typography)`
+  display: flex;
+
```

Comment:
why empty line are needed. I think your pritier is not working correctly (components also have issues with formating)